library/src/main/java/com/schibsted/spain/barista/internal/matcher/DrawableMatcher.kt

```diff
@@ -18,6 +18,9 @@ class DrawableMatcher(private val expectedId: Int) : TypeSafeMatcher<View>(View:
   companion object {
     private val EMPTY = -1
     private val ANY = -2
+
+    fun empty() = EMPTY
+    fun any() = ANY
```

Comment:
After doing what you requested, I think this code doesn't add value. I know that they are accessors to private constants but it doesn't worth the price IMHO. Just sayin'!

Reply:
You're right. Because that's not what I meant with factory methods 😂 

I mean:
```kotlin
companion object {
  private EMPTY=...
  private ANY=....
  
  fun any() = DrawableMatcher(ANY)
  fun empty() = DrawableMatcher(EMPTY)
  fun withId(@Idres id: Int) = DrawableMatcher(id)
```

It's the equivalent to the typical Java static methods to create the object. Think of the `createIntent(....)` pattern.

Reply:
@rocboronat I like what @Sloy suggests...
@Sloy you owe me one beer 🍺

Reply:
I like it too <3 Feel free to add code in your examples, to avoid misunderstanding your explanations. It will save time and effort. BTW, feel free to commit your suggestions to the PR. I will not discuss it, so let's simplify the development process.